opensfm/commands/debug_plot.py

```diff
@@ -708,11 +708,16 @@ def _rotation_matrix_to_euler_angles(R):
                                     float(delta_distance))
 
     culling_dict = {}
-    if os.path.exists('../frames_redundant/undo.txt'):
-        with open('../frames_redundant/undo.txt') as fin:
-            for line in fin:
-                (new_shot_id, orig_shot_id) = line.split()
-                culling_dict[new_shot_id] = orig_shot_id
+    if os.path.exists('../frames_filter.json'):
```

Comment:
what's our plan for moving this repo to a more tested state?

Reply:
That will probably involve some heavy refactoring (like splitting this 80 line "scoped under if main" block into separate functions, yeah? I guess it's okay to do then rather than now, if that's the plan.